src/Database/DBA.php

```diff
@@ -292,6 +292,20 @@ public static function insert($table, $param, $on_duplicate_update = false)
 		return DI::dba()->insert($table, $param, $on_duplicate_update);
 	}
 
+	/**
+	 * Replace a row of a table
```

Comment:
Please be more specific, something like
```suggestion
	 * Inserts a row with the provided data in the provided table.
	 * If the data corresponds to an existing row through a UNIQUE or PRIMARY index constraints, it updates the row instead.
```